src/libs/OptionsListUtils.js

```diff
@@ -409,8 +409,12 @@ function getLastMessageTextForReport(report) {
 
         // Yeah this is a bit ugly. If the latest report action that is not a whisper has been moderated as pending remove
         // then set the last message text to the text of the latest visible action that is not a whisper or the report creation message.
-        const lastNonWhisper = _.find(allSortedReportActions[report.reportID], (action) => !ReportActionUtils.isWhisperAction(action)) || {};
+        const lastNonWhisper =
+            _.find(allSortedReportActions[report.reportID], (action) => !ReportActionUtils.isWhisperAction(action) && !ReportActionUtils.isDeletedParentAction(action)) || {};
```

Comment:
`!ReportActionUtils.isDeletedParentAction(action)`

Why is this condition needed here?

Reply:
bump

Reply:
Why not use `shouldReportActionBeVisible` condition?
It will fix https://github.com/Expensify/App/pull/28002#issuecomment-1747402630 as well.

Reply:
Let's resole this last concern today. And then we're good

Reply:
can you test and confirm if it's really ignored?

`shouldReportActionBeVisible` should cover `[Deleted message]` case:
https://github.com/Expensify/App/blob/4078503d8e2468a3f0c54fc37af0f888ed2abf41/src/libs/ReportActionsUtils.js#L355